apps/vr-tests-react-components/src/stories/Menu/Menu.stories.tsx

```diff
@@ -97,6 +97,31 @@ storiesOf('Menu Converged - secondary content', module)
     { includeRtl: true },
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28675&commitId=21edc05e5364a91dbb6276ce3db52b610846090e&env=prod&clientType=FLUENTUI)


  <details><summary><b>Menu</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu.Scrollable Menu Small Viewport.default.chromium.png|184| Changed|

</details>


  <details><summary><b>Menu Converged - long content</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - long content.default - RTL.chromium.png|0| Added|
|Menu Converged - long content.default.chromium.png|0| Added|

</details>


  <details><summary><b>Menu Converged - split item</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - split item.default - RTL.chromium.png|215| Changed|
|Menu Converged - split item.default.submenu open.chromium.png|2081| Changed|
|Menu Converged - split item.default.chromium.png|215| Changed|
|Menu Converged - split item.default - RTL.submenu open.chromium.png|2084| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
The change in screenshot is because of border box style added to menuPopover.
Before: menuPopover is content box with minWidth 128 - which is for its content only (without padding). When `autoSize` is true, menuPopover is set to border box, so its minWidth is for content+10px padding.
After: menuPopover is border box with minWidth 138 always.